### PR TITLE
Changed formatting to comply with rust standards

### PR DESCRIPTION
**Pull request by sezna**
Went through and updated all the .rs files to comply with rust standard formatting. I figured I'd put it in a merge request instead, so that on the off chance that you don't like that format, you can reject it.

Also, sorry about the late start, had a busy week opening. I'm going to be much more active now.

I did a quick test compile and it should all be good!

### CODE REVIEW

**Comment by sezna** [comment]
running a  few more tests before I merge

**Comment by KalitaAlexey** [comment]
Your IDE brokes line ending.

**Comment by sezna** [comment]
Can you give me some example spots? I don't use an IDE.

**Comment by KalitaAlexey** [comment]
Main.rs

Sent from my Windows Phone

---

От: Alexmailto:notifications@github.com
Отправлено: ‎14.‎10.‎2015 9:51
Кому: slicebuild/sbmailto:sb@noreply.github.com
Копия: Alexeymailto:kalita.alexey@outlook.com
Тема: Re: [sb] Changed formatting to comply with rust standards (#9)

Can you give me some example spots? I don't use an IDE.

---

**Comment by sezna** [comment]
Sorry but I don't see anything wrong in main.rs - do you have some line numbers?

File: impl/rust/src/main.rs [hunk 1]
```diff
@@ -1,54 +1,64 @@
...
+                layer: layer,
+                os: os,
+                slice_root_directory: slice_root_directory,
+            })
+        }
+        _ => panic!(),
+    };
+}
+
+fn run_command<T>(mut command: T)
```
Comment (KalitaAlexey) [inline impl/rust/src/main.rs:52]: Opening paranthesis must be at end of line, isn't it?

**Pull request merged**